src/content/app/species/components/species-sidebar-modal/modal-views/SpeciesSidebarDownloads.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { useAppSelector } from 'src/store';
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
+
+import { useSpeciesFtpLinksQuery } from 'src/content/app/species/state/api/speciesApiSlice';
+
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+
+import styles from './SpeciesSidebarDownloads.module.css';
+
+const SpeciesSidebarDownloads = () => {
+  const activeGenomeId = useAppSelector(getActiveGenomeId) || '';
+  const { data: ftpLinksResponse } = useSpeciesFtpLinksQuery(activeGenomeId, {
+    skip: !activeGenomeId
+  });
```

Comment:
Seems like we would want a spinner while links are loading:

https://github.com/Ensembl/ensembl-client/assets/6834224/4c73b636-a403-42cd-a4f5-654371d65919

Check for `isFetching` field in the value returned from the `useSpeciesFtpLinksQuery` hook, and show the spinner while loading is in progress:

```
import { CircleLoader } from 'src/shared/components/loader';

...



  const { data: ftpLinksResponse, isFetching } = useSpeciesFtpLinksQuery(activeGenomeId, {
    skip: !activeGenomeId
  });

  if (isFetching) {
    return <CircleLoader />
  }
```
maybe also add a class to it that would center it as well.

_(there is a slight chance that Andrea would want us to put a special sidebar loader, like the one [here](https://staging-2020.ensembl.org/entity-viewer/grch38/variant:21:45988497:rs4647797?allele=0); but I hope she doesn't, because that might have implications for the title of the modal)_